CorsixTH/Lua/entities/machine.lua

```diff
@@ -237,7 +237,9 @@ function Machine:createHandymanActions(handyman)
   end
 
   local meander_loop_callback = --[[persistable:handyman_meander_repair_loop_callback]] function()
-    if not self.user then
+    -- Wait until the machine is not in use and not about to be used
+    local patient = self:getRoom():getPatient()
+    if not self.user and (not patient or patient.isLeaving) then
```

Comment:
My oversight, I think this one needs to be `patient:isLeaving()`

Calling `patient.isLeaving` just confirms that a function exists for the patient.